ProjectShop/Shipping/templates/all_shipping_addresses.html

```diff
@@ -0,0 +1,25 @@
+<ul>
+    <!-- Iterate over object_list -->
+    {% for object in object_list %}
+    <!-- Display Objects -->
+
+    <li><strong>Postal code: </strong> {{ object.postal_code }}</li>
```

Comment:
Translation to HTML files should be added.
Please, visit the link for more information.
https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#internationalization-in-template-code
Should you have any questions, feel free to contact me.